src/components/QuestionView/Expanded.tsx

```diff
@@ -50,12 +50,15 @@ export default function ExpandedView({
             <ActionIcon variant="transparent" title="Edit" onClick={setActiveQuestion}>
               <Icon name="pencil" width={22} />
             </ActionIcon>
-            <ActionIcon variant="transparent">
+            <ActionIcon variant="transparent" onClick={() => setExpandedQuestionId(null)}>
               <Icon name="caretUp" />
             </ActionIcon>
+            <ActionIcon variant="transparent" className="questionHandle">
```

Comment:
Do we need to drag in expanded mode? Dragging in expanded mode will be difficult.
Maybe we show it in collapsed mode only. And only show it after the user clicks the rearrange questions button and then we collapse all the questions. Does not make sense to show the drag icon all the time. Users might accidentally drag stuff.

Reply:
ok